scripts/jenkins-test

```diff
@@ -7,13 +7,15 @@ ADAM_MVN_TMP_DIR=$(mktemp -d -t adamTestMvnXXXXXXX)
 
 # add this tempdir to the poms...
 find . -name pom.xml \
-    -exec sed -i .bak \
+    -exec sed -i.bak \
     -e "s:sun.io.serialization.extendedDebugInfo=true:sun.io.serialization.extendedDebugInfo=true -Djava.io.tmpdir=${ADAM_MVN_TMP_DIR}:g" \
     {} \;
 find . -name "*.bak" -exec rm {} \;
 
 # variable declarations
-export JAVA_HOME=/usr/java/jdk1.8.0_60
+if [ -z "$JAVA_HOME" ]; then
```

Comment:
This is a bit ugly, but we always need to overwrite `$JAVA_HOME` here due to Jenkins doing "shenanigans" with the Java version.  The short story is that Jenkins will always set `$JAVA_HOME` to a bad value, and we are powerless to stop it. There's a much longer and uninteresting backstory. Can you revert this back, or add some other logic to disable overwriting `$JAVA_HOME`?